python/cudf/cudf/core/buffer.py

```diff
@@ -84,7 +84,7 @@ def _init_from_array_like(self, data):
 
     @classmethod
     def empty(cls, size):
-        dbuf = DeviceBuffer(ptr=None, size=size)
+        dbuf = DeviceBuffer(ptr=0, size=size)
```

Comment:
This feels a bit less Pythonic now, since `None` is much closer to `NULL` in Python land than `0`. But no strong objections.

Reply:
We could drop skipping setting the `ptr`. Would that be better?

Reply:
Yes - thanks :)

Reply:
Thanks for raising. Not sure why I didn't think of that before 😄